xml/System.Globalization/Calendar.xml

```diff
@@ -1059,6 +1059,8 @@
   
 [!INCLUDE[japanese-era-note](~/includes/calendar-era.md)]
 
+Of the calendar implementations in .NET, only the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar> classes support multiple eras. 
```

Comment:
>only the  [](start = 41, length = 9)

I would avoid telling "only" just in case in the future we can have more calendars have multiple eras. nobody knows :-)

Reply:
But if new calendar with multiple eras are added, I guess you'd have to rewrite this anyway.

Reply:
not really, the new calendars may not have anything related to Japanese calendars.